tests/CMakeLists.txt

```diff
@@ -138,21 +140,22 @@ endif()
 # INSTALLATION
 #----------------------------------
 
-set(TESTS_INSTALL_PATH "/opt/test/bin" CACHE STRING "Specifies where the test binaries will be installed")
+include(GNUInstallDirs)
+set(TESTS_INSTALL_PATH "tests" CACHE STRING "Specifies where the test binaries will be installed")
 
 install(TARGETS sdbus-c++-unit-tests DESTINATION ${TESTS_INSTALL_PATH} COMPONENT test)
 install(TARGETS sdbus-c++-integration-tests DESTINATION ${TESTS_INSTALL_PATH} COMPONENT test)
-install(FILES ${INTEGRATIONTESTS_SOURCE_DIR}/files/org.sdbuscpp.integrationtests.conf DESTINATION /etc/dbus-1/system.d COMPONENT test)
+install(FILES ${INTEGRATIONTESTS_SOURCE_DIR}/files/org.sdbuscpp.integrationtests.conf DESTINATION ${CMAKE_INSTALL_FULL_SYSCONFDIR}/dbus-1/system.d COMPONENT test)
```

Comment:
Just to make sure: You mentioned previously that `/etc/dbus-1` does not respect `CMAKE_INSTALL_PREFIX`, but this change doesn't respect it either. But I guess we don't want to respect it here, right?

Reply:
It does and confirmed by CI: integration tests on Ubuntu and FreeBSD would fail otherwise. `${CMAKE_INSTALL_FULL_SYSCONFDIR}` expands to `${CMAKE_INSTALL_PREFIX}/etc` or `/etc` if `${CMAKE_INSTALL_PREFIX}` is `/usr` (common on Linux).

Reply:
To see CI results enable workflow on this PR or for all PRs. I'm not gonna link to my results as I've removed `master` branch in my fork and plan to remove the whole fork once this is merged.